packages/frontend/editor-ui/src/app/css/_global.scss

```diff
@@ -1,9 +1,5 @@
-@use '@n8n/design-system/css/mixins' as ds-mixins;
-@use '@n8n/chat/css';
-@use '@/app/styles';
-
 :root {
-	// Using native css variable enables us to use this value in JS
+	--navbar--height: 64px;
```

Comment:
Nice cleanup of the `packages/frontend/editor-ui/src/app/styles/_global.scss` which only exposed this single CSS var